site/docs/pages/guides/contribution.mdx

```diff
@@ -65,6 +65,18 @@ onchainkit
 
 ## Workflows
 
+### Storybook
+
+Storybook is a frontend workshop for building UI components and pages in isolation. It helps you develop and share hard-to-reach states and edge cases without needing to run your whole app.
+
+To develop and test your components in Storybook, run the following command:
+
+```bash
+yarn storybook:dev
+
+# In a browser, navigate to http://localhost:6006/
```

Comment:
We can leave this out since `yarn storybook:dev` auto directs the user to that url